src/python/pants/backend/python/tasks/resolve_requirements_task_base.py

```diff
@@ -37,8 +37,8 @@ class ResolveRequirementsTaskBase(Task):
   def subsystem_dependencies(cls):
     return super(ResolveRequirementsTaskBase, cls).subsystem_dependencies() + (
       PexBuilderWrapper.Factory,
+      PythonSetup,
       PythonNativeCode.scoped(cls),
-      PythonSetup.scoped(cls),
```

Comment:
I think that this represents a breaking change... maybe one that we're ok with though.

A middle ground might be to leave behind the old `scoped` dep as deprecated (via `PythonSetup.scoped(cls, removal_version=.., removal_hint=..)`), and just not use it?:
https://github.com/pantsbuild/pants/blob/3742edb4040967899c404b43570caa721bb3bd95/src/python/pants/subsystem/subsystem.py#L52-L61